src/software/geom/util.cpp

```diff
@@ -1251,6 +1116,58 @@ std::optional<Segment> mergeFullyOverlappingSegments(Segment segment1, Segment s
     }
 }
 
+std::vector<Segment> reduceParallelSegments(std::vector<Segment> segments)
+{
+    // If there are no segments, return nullopt
```

Comment:
comment is out-of-date now I think? Also fairly redundant, it's pretty clear what we're doing here, so I'd just remove it.

Reply:
Removed